replace-namespace.sh

```diff
@@ -19,3 +18,5 @@ find "$SCRIPT_DIR" -type f \( -name "*.php" -o -name "*.json" \) ! -name "$SCRIP
 	# Use perl for the replacement in each file
 	perl -pi -e "s/WP_Compat_Validation_Tool/$NEW_NAMESPACE/g" "$file"
 done
+
+cd 10up-lib/wp-compat-validation-tool && rm -rf .github .gitignore composer.json composer.lock CHANGELOG.md CONTRIBUTING.md README.md LICENSE.md CODE_OF_CONDUCT.md CREDITS.md
```

Comment:
@Sidsector9 should we use `*.md` and `*.lock` to cover all files instead of hard coding each file name?

Reply:
Yeah, I was being very careful with each file 😅 
I'll handle this in the next PR.